### PR TITLE
Created LocalizationTest

### LINKED ISSUE #9: Add an example how to use the neodymium localization feature
Showcase the localization feature by calling the xceptance homepage in german and english.
Add validation of a specific WebElement that has different wordings according to the translation.

Add this to the documentation within the Neodymium wiki.

### PR DESCRIPTION
**Pull request by georgkunze**
Created initial localization test.
#9

### CODE REVIEW

File: config/localization.yaml [hunk 1]
```diff
@@ -1,4 +1,10 @@
 default:
   homepage:
     title: Xceptance - The Software Testing Experts 
+    headline: Committed to Software Quality
+
```
Comment (occupant23) [inline config/localization.yaml:6]: use german locale 'de'

File: src/test/java/showcase/neodymium/tests/localization/LocalizationTest.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        headLine.shouldBe(visible);
+        
+        //Check that headline is correct in english
+        headLine.shouldHave(text(Neodymium.localizedText("homepage.headline")));
+        
+        Selenide.closeWindow();
+        
+        //Open homepage in german
+        Selenide.open("https://www.xceptance.com/de/");
+
```
Comment (occupant23) [inline src/test/java/showcase/neodymium/tests/localization/LocalizationTest.java]: change locale by manipulating the Neodymium property
`Neodymium.configuration().setProperty("neodymium.locale", "de");`

File: src/test/java/showcase/neodymium/tests/localization/LocalizationTest.java [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+        Selenide.closeWindow();
+        
+        //Open homepage in german
+        Selenide.open("https://www.xceptance.com/de/");
+        
+        //Check title is correct in german
+        //Note: To use localized string with locale you have to call them like this:
+        //Neodymium.localizedText(key-to-string, locale)
+        //This will return the translated string for this locale, if it exists.
+        new Title().validateTitle(Neodymium.localizedText("homepage.title", "de"));
```
Comment (occupant23) [inline src/test/java/showcase/neodymium/tests/localization/LocalizationTest.java]: if locale is change the second parameter is not needed

File: src/test/java/showcase/neodymium/tests/localization/LocalizationTest.java [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+        
+        //Headline example
+        
+        //Check headline element is visible        
+        headLine.shouldBe(visible);
+        
+        //Check that headline is correct in german
+        headLine.shouldHave(text(Neodymium.localizedText("homepage.headline", "de")));
```
Comment (occupant23) [inline src/test/java/showcase/neodymium/tests/localization/LocalizationTest.java]: if locale is change the second parameter is not needed

File: src/test/java/showcase/neodymium/tests/localization/LocalizationTest.java [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+    }
```
Comment (occupant23) [inline src/test/java/showcase/neodymium/tests/localization/LocalizationTest.java:87]: add another localization item under "german" and demonstrate the specific call for a locale, add the same element for the english part above.

**Review by occupant23 — CHANGES_REQUESTED** [review]

File: src/test/java/showcase/neodymium/tests/localization/LocalizationTest.java [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+        //Check headline element is visible        
+        headLine.shouldBe(visible);
+        subTitle.shouldBe(visible);
+        
+        //Check that headline is correct in german
+        headLine.shouldHave(text(Neodymium.localizedText("homepage.headline")));
+        
+        //Check that subtitle is correct in german
+        //Note: Here the call to localizedText has a added locale to get a specific String
+        //from a specific locale.
```
Comment (ckeiner) [inline src/test/java/showcase/neodymium/tests/localization/LocalizationTest.java]:
```suggestion
        //Here the call to localizedText has another locale as second parameter to get
        //a specific String from that specific locale.
```

IMO, this sounds more concise.

**Review by ckeiner — CHANGES_REQUESTED** [review]

**Review by ckeiner — COMMENTED** [review]
I will approve of these changes once occupant23's unresolved comment is completed.

**Comment by georgkunze** [comment]
@ckeiner i added a check for the subtitle for the english site.

**Review by ckeiner — APPROVED** [review]

**Pull request merged**